CactbotOverlay/CactbotOverlay.cs

```diff
@@ -247,6 +253,90 @@ public CactbotOverlay(CactbotOverlayConfig config)
       fast_update_timer_semaphore_.Release();
     }
 
+    public void EnableWatchers(bool check)
+    {
+      if (!init_ || this.watchers == null)
+      {
+        this.watcherState = check;
+        return;
+      }
+      // I doubt that the following code is actually necessary
+      if (this.watchers.Length > 0)
+        foreach (var watcher in this.watchers)
+          watcher.EnableRaisingEvents = check;
+    }
+ 
+    private void SetupFileWatcher(string url)
+    {
+      if (!init_)
+        return;
+      if (this.watchers != null && this.watchers.Length > 0)
+        foreach(var watcher in this.watchers)
+          watcher.Dispose();
+      List<string> pathsList = new List<string>();
+      
+      // Overlay Dir
+      if (url == "")
+        return;
+
+      var overlayFullPath = url;
+      overlayFullPath = Regex.Replace(overlayFullPath, @"file:[\\\/]+", "");
+      var overlayPath = Path.GetDirectoryName(url);
```

Comment:
This doesn't work when your url is remote, e.g. `https://quisquous.github.io/cactbot/ui/raidboss/raidboss.html`

Reply:
Ouch, yeah didn't think of that